src/pres/loader/Source.js

```diff
@@ -64,7 +72,7 @@ export default class Source extends React.Component {
             expanded={this.props.expandedPanel === 'source'}
             onChange={this.props.handleExpansionChange}>
             <AccordionSummary
-                expandIcon={<ExpandMoreIcon />}
+                expandIcon={<ExpandMore />}
```

Comment:
The icon name needed updating so that it could be imported like `import { ExpandMore } from '@material-ui/icons';`